csrc/kernel_cache.cpp

```diff
@@ -337,7 +406,12 @@ void FusionExecutorCache::evictCache(size_t cache_id) {
 
 FusionKernelRuntime* FusionExecutorCache::getKernelRuntimeFor(
     const KernelArgumentHolder& args,
-    std::optional<PrimDataType> forced_index_type) {
+    std::optional<PrimDataType> forced_index_type,
+    Fusion* fusion) {
```

Comment:
so instead of passing a fusion pointer here, we can just have concretizationInfo here.